### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by andyk-ms**
Adding Sophia

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Track1 .NET Azure IoT Hub and DPS SDKs
 
-*	@drwill-ms @timtay-microsoft @abhipsaMisra @andyk-ms @brycewang-microsoft @tmahmood-microsoft @ngastelum-ms @patilsnr
+*	@drwill-ms @timtay-microsoft @abhipsaMisra @andyk-ms @brycewang-microsoft @tmahmood-microsoft @ngastelum-ms @patilsnr @schomims
```
Comment (schoims) [inline .github/CODEOWNERS:3]: there is a typo. my id is @schoims

**Review by schoims — CHANGES_REQUESTED** [review]

**Review by schoims — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (andyk-ms) [reply .github/CODEOWNERS]: updated.

**Review by timtay-microsoft — APPROVED** [review]

**Review by brycewang-microsoft — APPROVED** [review]

**Pull request merged**